docs/docs/pricing/README.md

```diff
@@ -9,137 +9,150 @@ We believe anyone should be able to run simple, low-volume workflows at no cost.
 
-The Professional Tier includes all the features of the Developer Tier. It also comes with the following benefits:
+[Visit our pricing page](https://pipedream.com/pricing) to learn more about our paid plan options.
 
-- **You have no daily invocations or compute time cap. You can run any number of invocations, for any amount of time**.
-- Your event history per workflow increases from {{$site.themeConfig.FREE_INSPECTOR_EVENT_LIMIT}} to {{$site.themeConfig.PAID_INSPECTOR_EVENT_LIMIT}}.
-- You can increase the [max time per execution](/limits/#time-per-execution) to 750 seconds (up from 300 on the Developer Tier).
-- You can request [QPS increases](/limits/#qps-queries-per-second) for specific HTTP endpoints.
-- When using [concurrency and throttling controls](/workflows/concurrency-and-throttling/), you can increase a workflow's queue size up to {{$site.themeConfig.MAX_WORKFLOW_QUEUE_SIZE}} (free users are capped at a queue size of {{$site.themeConfig.DEFAULT_WORKFLOW_QUEUE_SIZE}}).
+All paid plans vary features based on tier, but each paid plan option will:
 
-### Upgrading to the Professional Tier
+* Lift the daily 100 credits limit
+* Increase the number of active workflows available
+* Increase the number of connected accounts
 
-You can upgrade to the Professional Plan by visiting your [Billing and Usage Settings](https://pipedream.com/settings/billing).
+## Definition of Terms
 
-### Professional Tier Pricing
+Pipedream uses a number of terms to describe platform metrics and details of our plans. See the definitions of key terms below.
 
-See [https://pipedream.com/pricing](https://pipedream.com/pricing) for pricing information.
+### Credits
 
-### Professional Tier Support options
+::: tip
 
-Users on the Professional Tier have access to community support, on [our forum](https://pipedream.com/community) and Slack. [Visit out Support page](https://pipedream.com/support) for more information.
+If you're still on our grandfathered Developer, Professional or Organization plan, you still may be incurring **invocations** and not credits.
 
-## Team Plan
+During this transitionary period, you'll still be billed in terms of invocations. However workspace migrations are coming. You'll be guided on how to migrate your account to a [workspace](/docs/workspaces/) in the near future.
 
-You can create as many [organizations](/orgs/) as you'd like for free. But the quota for free orgs is set to {{$site.themeConfig.FREE_ORG_DAILY_INVOCATIONS_LIMIT}} invocations per day. To lift this limit, you can upgrade the org to the **Team Plan**.
+:::
 
-Orgs are limited to {{$site.themeConfig.TEAM_MEMBER_LIMIT}} members. If you need more than {{$site.themeConfig.TEAM_MEMBER_LIMIT}} team members, please [reach out](https://pipedream.com/support).
+Pipedream charges one credit per {{ $site.themeConfig.base_credits_price.seconds }} seconds of compute time at {{ $site.themeConfig.base_credits_price.memory }} megabytes of memory (the default) per workflow execution.
```

Comment:
Maybe just to be super clear to users here, you can add a sentence like: "If you keep the default memory settings, and your workflow executions run under 30 seconds, you'll incur 1 credit per run."